contracts/src/Teleporter/tests/TeleporterMessengerTest.t.sol

```diff
@@ -299,4 +299,10 @@ contract TeleporterMessengerTest is Test {
                 payload: payload
             });
     }
+
+    function _formatErrorMessage(
```

Comment:
Can we rename this and other similar functions to `_formatTeleporterErrorMessage`? That way it's clear at the callsite what function is being called.